lib/rubocop/cop/correctors/parentheses_corrector.rb

```diff
@@ -22,6 +26,30 @@ def ternary_condition?(node)
         def next_char_is_question_mark?(node)
           node.loc.last_column == node.parent.loc.question.column
         end
+
+        def only_closing_paren_before_comma?(node)
+          source_buffer = node.source_range.source_buffer
+          line_range = source_buffer.line_range(node.loc.end.line)
+
+          /^\s*\)\s*,/.match?(line_range.source)
```

Comment:
Can you use `start_with?`?

```suggestion
          line_range.source.start_with?(/\s*\)\s*,/)
```

Reply:
Updated!